Makefile

```diff
@@ -32,7 +33,7 @@ luacheck:
 
 .PHONY: test
 test:
-	luatest -v --coverage
+	luatest -v --coverage --shuffle all:${SEED} --seed ${SEED}
```

Comment:
We can pass it once: https://github.com/tarantool/luatest/blob/f639ce4c75808166163c60b50e88a7a942b3b885/luatest/runner.lua#L137-L144.